.github/actions/set_github_user/action.yml

```
@@ -0,0 +1,9 @@
+name: Set GitHub User
+description: 'Sets github user for a particular workflow'
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        git config user.name paypalsdk
+        git config user.email sdks@paypal.com
```

I think we decided we should use braintreeps for btsdks and paypalsdk for unified sdk

```suggestion
        git config user.name braintreeps
        git config user.email code@getbraintree.com
```